apps/main-app/src/components/paralogy/paralogyTable.js

```diff
@@ -0,0 +1,122 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import MethodHeader from '../orthology/methodHeader';
+import MethodCell from '../orthology/methodCell';
+import BooleanCell from '../orthology/booleanCell';
+import {
+  getOrthologSpeciesName,
+  getOrthologId,
+  getOrthologSymbol,
+} from '../orthology/utils';
+import HelpPopup from '../helpPopup';
+
+import style from '../orthology/style.scss';
+
+const columns = [
+  {name: 'Gene symbol'},
+  {name: 'Count'},
+  {name: 'Best', help: <span>This gene is called a paralog of the input gene by the highest number of algorithms.<br/> In specific cases, ZFIN curators have asserted reliable paralogy to the gene called by the second highest number of algorithms. These are denoted <strong>Yes *</strong>.</span>},
+  {name: 'Best reverse', help: 'The input gene is called a paralog of this gene by the highest number of algorithms.'},
+  {name: 'Method'}
+];
+
+export function isBest(value = '') {
+  return typeof value === 'boolean' ? value : value.match(/yes/i);
+}
+
+class ParalogyTable extends Component {
+
+  render() {
+    let rowGroup = 0;
+    return(
+      <table className='table'>
+        <thead className='text-capitalize'>
+          <tr>
+            {
+              columns.map((column) => {
+                if (column.name === 'Method') {
+                  return (<MethodHeader key={column.name} name={column.name} />);
+                } else {
+                  return (<th key={column.name}>
+                    {column.name} {column.help && <HelpPopup id={`help-${column.name}`}>{column.help}</HelpPopup>}
+                  </th>);
+                }
+              })
+            }
+          </tr>
+        </thead>
+        <tbody>
+          {
+            this.props.data.map((orthData, idx, orthList) => {
+              const scoreNumerator = orthData.predictionMethodsMatched.length;
+              const scoreDemominator = scoreNumerator +
+                orthData.predictionMethodsNotMatched.length;
+              const orthId = getOrthologId(orthData);
+
+              if (idx > 0 && getOrthologSpeciesName(orthList[idx - 1]) !== getOrthologSpeciesName(orthData)) {
+                rowGroup += 1;
+              }
+
+              return (
+                <tr className={rowGroup % 2 === 0 ? style.groupedRow : ''} key={orthId}>
+                  <td>
+                    <Link to={`/gene/${orthId}`}>
+                      <span dangerouslySetInnerHTML={{__html: getOrthologSymbol(orthData)}} />
```

Comment:
Can that method be called getHomologSymbol?

Reply:
I was thinking in creating a follow up ticket to refactor the orthology code called here and make it generic (renaming to homologSomething) to call it from orthology and paralogy

Reply:
renamed, but leaving orthology code untouched